### ISSUE TITLE
Moving together column groups

### ISSUE THREAD

**Issue by aditya-thimphu** [olifolkerd/tabulator#655]
Hi Oli,

Tried moving column groups (column headers grouped and movableColumns: true), but failed. I need a feature to move a group of columns together. Your comments or advise please.

Thanks,
Aditya

**Comment by olifolkerd** [comment]
Hey @aditya-thimphu 

Thanks for getting in touch, at the moment column groups are not movable in the same way as columns, but i think that is a great idea and ive added it to the roadmap for inclusion in a later release.

Thanks for the suggestion

Oli

**Comment by aditya-thimphu** [comment]
Thanks Oli. I look forward to the new release.

**Comment by billsymo** [comment]
Oli,

I noticed this feature is slated for release 4.3 on your trello board.  Is there an ETA for that?

**Comment by olifolkerd** [comment]
Hey @billsymo 

I generally do a minor version release every 2-3 months so probably around May.

Cheers

Oli :)

**Comment by Shreyas-BS** [comment]
Hi any updates on this feature???

**Comment by olifolkerd** [comment]
Nope, nothing as yet, this hasn't been allocated to a specific release yet.

There is going to be a small release this month, then all effort is being focused on the 5.0 rebuild,

After that I will look at building out a plan for the next few releases

Cheers

Oli :)

**Comment by jlm0083** [comment]
Hi Oli,

Is there any chance to allocate this enhancement to a recent release?

Thanks,
Julian
